client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteOf } from 'state/selectors';
+import { isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+
+/**
```

Comment:
I'm not sure I understand the use case this selector would be needed in - can you please explain why we need this selector and we can't just use just `isJetpackSiteSecondaryNetworkSite`?